src/main/java/com/amplitude/api/IdentifyInterceptor.java

```diff
@@ -105,7 +105,7 @@ private JSONObject getTransferIdentifyEvent() {
             identifyEvent.getJSONObject("user_properties").put(Constants.AMP_OP_SET, identifyEventUserProperties);
             dbHelper.removeIdentifyInterceptors(lastIdentifyInterceptorId);
             return identifyEvent;
-        } catch (JSONException e) {
+        } catch (Exception e) {
```

Comment:
I have started using `Throwable` which covers `Exception`, `Error` and `Violation` - https://developer.android.com/reference/java/lang/Throwable

```suggestion
        } catch (Throwable e) {
```